### PR TITLE
Jan SDK - Setup and test the demo app

### PR DESCRIPTION
**Pull request by UdayaNalla**
## Description

Jan SDK release - Setup and test the demo app

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

File: AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec [hunk 1]
```diff
@@ -8,7 +8,7 @@ Pod::Spec.new do |s|
 
   s.author           = { 'Microsoft' => 'appcentersdk@microsoft.com' }
 
-  s.source = { :http => "https://github.com/Microsoft/AppCenter-SDK-React-Native/releases/download/#{s.version}/AppCenter-SDK-ReactNative-iOS-Pod-#{s.version}.zip" }
+  s.source = { :http => "https://mobilecentersdkdev.blob.core.windows.net/sdk/AppCenter-SDK-ReactNative-iOS-Pod-#{s.version}.zip" }
```
Comment (guperrot) [inline AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec]: @dhei is this a temporary change? Is this accidental change?

**Comment by UdayaNalla** [comment]
TGZ files have been deleted, will research on the squash merging strategy and use that moving forward

**Comment by guperrot** [comment]
@UdayaNalla this is just when you merge on github ui instead of just clicking merge, you can select another strategy in the menu, I can make a screenshot after approval.

File: AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec (hunk 1 above)
Comment (guperrot) [inline AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec]: Revert this change. The public pod spec must use github.

Comment (guperrot) [inline AppCenterReactNativeShared/Products/AppCenterReactNativeShared.podspec]: Revert this change. The public pod spec must use github.

**Comment by guperrot** [comment]
@UdayaNalla the release procedure was updated to avoid the accidental change on podspec.
